ios-base.xcodeproj/project.pbxproj

```diff
@@ -23,6 +23,10 @@
 		074D20D9248E993B002A39B4 /* AuthenticationServices.swift in Sources */ = {isa = PBXBuildFile; fileRef = 074D20D8248E993B002A39B4 /* AuthenticationServices.swift */; };
 		074D20DB248EA832002A39B4 /* UserServices.swift in Sources */ = {isa = PBXBuildFile; fileRef = 074D20DA248EA832002A39B4 /* UserServices.swift */; };
 		07741D72218CDED600DB3B97 /* FirstViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 07741D71218CDED600DB3B97 /* FirstViewModel.swift */; };
+		241E32812A664B4D00B61972 /* Swifter in Frameworks */ = {isa = PBXBuildFile; productRef = 241E32802A664B4D00B61972 /* Swifter */; };
+		241E32832A664B7E00B61972 /* FacebookCore in Frameworks */ = {isa = PBXBuildFile; productRef = 241E32822A664B7E00B61972 /* FacebookCore */; };
```

Comment:
This FacebookCore dependency must be deleted. Please make sure the rebase against the main branch did not leave any reference behind.